code-scanning/properties/zscan.properties.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "zScan",
+    "creator": "Zimperium",
+    "description": "The zimperium-zscan GitHub action scans your mobile app binary (iOS or Android) and identifies security, privacy, and compliance-related vulnerabilities. ​",
+    "iconName": "zScan",
```

Comment:
Can you lower cap the `S`? Seems like it is failing the check

Reply:
I renamed the iconName reference to "zscan"